packages/stencil/src/components/datacom-avatar/datacom-avatar.tsx

```diff
@@ -0,0 +1,46 @@
+import { Component, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'datacom-avatar',
+  styleUrl: 'datacom-avatar.css',
+  shadow: true,
+})
+export class DatacomAvatar {
+  /* Custom properties */
+  @Prop() firstName: string;
+  @Prop() lastName: string;
+  @Prop() jobTitle: string;
+  @Prop() companyName: string;
+  @Prop() src: string;
+
+  render() {
+    let avatarImage;
+
+    if (this.src?.length > 0) {
+      avatarImage = <img alt="Employee Image" src={this.src} />;
```

Comment:
Considering this image is mostly decorative, you probably don't need the alt tag you're supplying here? Also, "Employee Image" is quite specific. There might be the possibility we could use this in other contexts not related to employees.

If you think it absolutely needs to be there, then maybe you can make the "alt" value a prop for this component, so that it's freely configurable.

Let me know if you disagree or not.

Reply:
Hi Angel, 
I was unable to save the file without error if I don't include the alt attribute. 
I will add it as a new prop for this component as suggested.